canonical-web-and-design/jaas.ai
# Update Prettier and add eslint linting

## Done

Update Prettier and add Eslint linting

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Sanity check code
- Ensure all tests pass
